client/keys/add.go

```diff
@@ -253,44 +252,20 @@ func runAddCmd(_ *cobra.Command, args []string) error {
 }
 
 func printCreate(info keys.Info, showMnemonic bool, mnemonic string) error {
-	output := viper.Get(cli.OutputFlag)
-
-	switch output {
-	case OutputFormatText:
-		fmt.Fprintln(os.Stderr)
-		printKeyInfo(info, keys.Bech32KeyOutput)
-
-		// print mnemonic unless requested not to.
-		if showMnemonic {
-			fmt.Fprintln(os.Stderr, "\n**Important** write this mnemonic phrase in a safe place.")
-			fmt.Fprintln(os.Stderr, "It is the only way to recover your account if you ever forget your password.")
-			fmt.Fprintln(os.Stderr, "")
-			fmt.Fprintln(os.Stderr, mnemonic)
-		}
-	case OutputFormatJSON:
-		out, err := keys.Bech32KeyOutput(info)
-		if err != nil {
-			return err
-		}
-
-		if showMnemonic {
-			out.Mnemonic = mnemonic
-		}
+	out, err := keys.Bech32KeyOutput(info)
+	if err != nil {
+		return err
+	}
 
-		var jsonString []byte
-		if viper.GetBool(client.FlagIndentResponse) {
-			jsonString, err = cdc.MarshalJSONIndent(out, "", "  ")
-		} else {
-			jsonString, err = cdc.MarshalJSON(out)
-		}
+	if showMnemonic {
+		out.Mnemonic = mnemonic
+	}
 
-		if err != nil {
-			return err
-		}
-		fmt.Fprintln(os.Stderr, string(jsonString))
-	default:
-		return fmt.Errorf("I can't speak: %s", output)
+	jsonString, err := cdc.MarshalJSONIndent(out, "", strings.Repeat(" ", viper.GetInt(client.FlagIndent)))
```

Comment:
I'm not sure `MarshalJSONIndent` with 0 indent will produce the same output as `MarshalJSON`, it should be a single line to be parsable everywhere